test/plugins/features/movies/controller.test.js

```diff
@@ -39,4 +39,65 @@ describe('movie controller', () => {
 
   });
 
+  describe('list', () => {
+
+    it('should properly sort based on query params', () => {
+      const query = { sort: 'release_year' , q: '' };
+      const reverseQuery = { sort: '-release_year', q: '' };
+
+      Controller.list(query)
+      .then((movies) => {
+        let ordered = true;
+
+        movies.each((movie, index) => {
+          if (!movie.release_year || !movies[index + 1].release_year) {
+            return;
+          }
+
+          if (movie.release_year < movies[index + 1].release_year) {
+            ordered = false;
+          }
+        });
```

Comment:
one trick might be to sort movies by release_year, and then compare it with the original to tell if it was sorted in the first place. might save some lines, but this works great too.